src/Manager.java

```diff
@@ -0,0 +1,81 @@
+
+public class Manager {
+	
+	public static void main(String args[])
+	{
+		int turns = 1;
+		
+		boolean isCaught = false;
+		Hare hare = new Hare(0,0);
+		Hound hound = new Hound(0,0);
+		hound.setDirection(hare.getDirection());
+		while(!isCaught && turns <= 100)
+		{
+			System.out.println("Turn:" +turns);
+			if(turns <=3)
+			{
+				hare.move(3, hare.getDirection());
+			}
+			else 
+			{
+				if(Math.sqrt(Math.pow(getXDisplacement(hare,hound), 2) + Math.pow(getYDisplacement(hare,hound), 2)) <=2)
+				{
+					isCaught = true;
+					break;
+				}
+					
+				if(turns % 3 == 0)
+				{
+					hare.setDirection(Directions.randomDirection());
+				}
+				if(turns % 5 == 0)
+				{
```

Comment:
I would create a separate method for the body of this if statement, something like findBestDirection(), then pass that result to hound.setDirection(). Maybe consider making it a method of Hound instead since this is hound-specific logic.